src/PrototypeRouteListener.php

```diff
@@ -117,9 +119,15 @@ public function onMergeConfig(ModuleEvent $e)
                     . $config['router']['routes'][$routeName]['options']['route'];
             }
 
+            $routeVersion = $this->versionRouteOptions;
+            if (isset($config['zf-versioning']['default_version']) &&
```

Comment:
As I noted before the last `isset` check is sufficient and the top one can be removed.

Reply:
ouch! done now!